src/components/common/GravatarWithTooltip.js

```diff
@@ -0,0 +1,64 @@
+import React, { Component } from 'react';
+import Gravatar from 'react-gravatar';
+import { Tooltip } from 'reactstrap';
+
+class GravatarWithTooltip extends Component {
+  constructor(props) {
+    super(props);
+    this.toggle = this.toggle.bind(this);
+    this.state = {
+      tooltipOpen: false,
+      showTooltip: typeof this.props.showTooltip !== 'undefined' ? this.props.showTooltip : true
```

Comment:
why don't just
```js
showTooltip: props.showTooltip,
```
also you can access props without `this` because its an argument in constructor